misc/package/Data Files/MWSE/core/mcm/main.lua

```diff
@@ -94,12 +169,39 @@ local function onClickCloseButton(e)
 	end
 end
 
---- @param a string
---- @param b string
-local function caseInsensitiveSorter(a, b)
-	return a:lower() < b:lower()
+--- Callback for when the favorite button has been clicked.
+--- @param e tes3uiEventData
+local function onClickFavoriteButton(e)
+
+
+	-- `source` is the button, which is right of the mod name, so we need to to up and then down-left
+	local package = configMods[e.source.parent.children[1].text]
+	package.favorite = not package.favorite
+	
+	updateFavoriteImageButton(e.source, package.favorite)
+
+
+
+
+
+	local menu = tes3ui.findMenu("MWSE:ModConfigMenu")
+	if not menu then return end
+	local modList = menu:findChild("ModList")
+	local modListContents = modList and modList:getContentElement()
+	
+	if not modListContents then 
+		tes3.messageBox("error! modlistcontents not found.")
```

Comment:
Won't be of much use to us if this condition is ever reached on the user's machine if it's a messagebox. Can you log the error?